ui/util.py

```diff
@@ -107,3 +119,31 @@ def get_expiration(query_params, default_duration=timedelta(hours=2)):
             return datetime.utcnow() + parsed
 
     return datetime.utcnow() + default_duration
+
+
+def get_et_job(job_id):
+    """
+    Get the status of an ElasticTranscode job
+
+    Args:
+        job_id(str): ID of ElasticTranscode Job
+
+    Returns:
+        JSON representation of job status/details
+    """
+    et = boto3.client('elastictranscoder', settings.AWS_REGION)
```

Comment:
Should probably refactor `boto3.client('elastictranscoder', settings.AWS_REGION)` to a method since it's used in a few places.